zeusd/src/devices/gpu/linux.rs

```diff
@@ -25,9 +25,21 @@ impl NvmlGpu<'static> {
 }
 
 impl GpuManager for NvmlGpu<'static> {
+    // TODO: Implementation wil squash NVML errors that users have to be aware of.
```

Comment:
I think this should be addressed before we merge this PR.

Reply:
I think catching the specific error that is raised when no GPUs are available would fix it. From [NvmlError docs](https://docs.rs/nvml-wrapper/latest/nvml_wrapper/error/enum.NvmlError.html), matching with `Err(NvmlError::LibloadingError(e))` works.

Reply:
Sounds good!